CodeStarterCamp_Week4/BodyCondition.swift

```diff
@@ -2,34 +2,40 @@
 //  BodyCondition.swift
 //  CodeStarterCamp_Week4
 //
-//  Created by Whales on 2023/03/17.
+//  Created by Whales on 2023/03/20.
 //
 
-import Swift
+import Foundation
 
 class BodyCondition {
-    var upperBodyMuscleStrength: Int {
+    var upperBodyMuscleStrength: Int{
```

Comment:
```suggestion
    var upperBodyMuscleStrength: Int {
```
아래도 프로퍼티 감시자 컨벤션이 통일되지 않네요!

Reply:
아...제가 저번꺼 수정 안 한 걸로 작업을 했더니 또ㅠㅠㅠ 웬수가 또... ( •᷄⌓•᷅ )  ㅠㅠㅠㅠㅠㅠㅠ
이것도 아래꺼도 다 수정해놓겠습니다ㅠㅠㅠ